keyboards/bm60rgb_iso/bm60rgb_iso.c

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2020 markva
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "bm60rgb_iso.h"
+
+led_config_t g_led_config = { {
```

Comment:
We'd like this wrapped in an `#if defined`, because without it this keyboard fails to compile in the event a user wants to disable the RGB.

```suggestion
#if defined(RGB_MATRIX_ENABLE)
led_config_t g_led_config = { {
```